docs/install/_linux.rst

```diff
@@ -136,6 +109,35 @@ To change the system service owner, you need to change the configuration of the
 .. note:: Replace the ``$USER`` in commands above with the name of the user you wish to be the new Kolibri system service owner.
 
 
+Higher Performance with the ``kolibri-server`` package
+------------------------------------------------------
+
+We have of a higher-performance wrapper around the ``kolibri`` package for Ubuntu and Debian systems called ``kolibri-server``. This version is recommended for:
+
+* Clear server-client setup for implementations with many client devices
+* When the server device has more than one CPU core
+
+``kolibri-server`` has some specific features like **static content cache** and **optimized usage of the CPU processing power** in multi core devices. Typical use case would be a *Raspberry Pi 3+* device with 4 CPU cores, which will perform much better and faster with ``kolibri-server`` compared to base Kolibri.
```

Comment:
I think we should be bold enough to be very clear about the real benefits of using `kolibri-server`:

```suggestion
``kolibri-server`` has some specific features like **static content cache** and **optimized usage of the CPU processing power** in multi core devices. For example, a *Raspberry Pi 3+* device has 4 CPU cores, and with ``kolibri-server`` it is likely to serve 2-3x as many clients.
```

Reply:
Added your suggestion! :slightly_smiling_face: